updated Language docs for mobile, created one for Web (#766)
MapsPeople/docs: src/shared/display-language/cms-language.md

## Configuring POI translations in CMS

> To provide multiple languages for items in the MapsIndoors CMS, such as "Meeting Room" or "Restroom", the translation must be provided by the user in the CMS. A translation can be provided in any language. In order to add non-default languages, please contact your MapsIndoors representative, and we will enable you to add translations in your desired language.

Reviewer: ```suggestion
To provide multiple languages for items in the MapsIndoors CMS, such as "Meeting Room" or "Restroom", the translation must be provided by the user in the CMS. A translation can be provided in any language. In order to add support for additional languages that we currently do not support, please contact your MapsIndoors representative, and we will enable you to add translations in your desired language.
```

> To provide multiple languages for items in the MapsIndoors CMS, such as "Meeting Room" or "Restroom", this translation must be provided by the user in the CMS. A translation can be provided in any language, and any pre-defined phrase will default to English. In order to add non-default languages, please contact your MapsIndoors representative, and we will enable you to add translations in your desired language.

Reviewer: ```suggestion
To provide multiple languages for items in the MapsIndoors CMS, such as "Meeting Room" or "Restroom", the translation must be provided by the user in the CMS. A translation can be provided in any language, and any pre-defined phrase will default to English. In order to add non-default languages, please contact your MapsIndoors representative, and we will enable you to add translations in your desired language.
```

Reviewer: Hi @JG1995 

Can you elaborate on what "any pre-defined phrase will default to English" means?

Author: I think that's a previous wording from while I was still barely understanding it, I'll reword or just remove.
